Checks/PWR004/README.md

```diff
@@ -52,6 +54,39 @@ void example() {
 }
 ```
 
+#### Fortran
+
+In the following code, a variable `factor` is used in each iteration of the
+loop to initialize the array `result`:
+
+```f90
+subroutine example()
+  integer :: factor = 42
+  integer :: result(10)
+
+  !$omp parallel do
+  do i = 1, 10
+    result(i) = factor * i
+  end do
+end subroutine example
+```
+
+Having the scope declared explicitly for each variable improves readability,
+since it makes explicit the scope of all the variables within the parallel
+region:
+
+```f90
+subroutine example()
+  integer :: factor = 42
+  integer :: result(10)
+
+  !$omp parallel do shared(factor, result)
```

Comment:
I fully agree with your suggestion. Especially considering that this documentation should be taken as a learning resource, so it's better to be as explicit as possible for the sake of learning, instead of relying on OpenMP's implicit behaviors.

Thank you for the suggestion! Should be [done](https://github.com/codee-com/open-catalog/compare/70d218ee94a154404de7f2047bac58ee0689635a..53da9a7b92e35919f1543c12f7650952d5c94324).